api/src/server.ts

```diff
@@ -3,8 +3,24 @@ import cors from "cors";
 import * as notangles from "./notangles/index";
 import * as freerooms from "./freerooms/index";
 import { writeData } from "./write-data";
+import * as Sentry from "@sentry/node";
+import * as Tracing from "@sentry/tracing";
 
 const app = express();
+
+// Sentry configurations
+Sentry.init({
+    dsn: `${process.env.INGEST_URL}`,
+    integrations: [
+        new Sentry.Integrations.Http({ tracing: true }),
+        new Tracing.Integrations.Express({ app }),
+    ],
+    // might need to change this for production
+    tracesSampleRate: 0.6,
```

Comment:
If this is something that can potentially change in production, set it to an environment variable as well please.